test/Microsoft.ML.Tests/TextClassificationTests.cs

```diff
@@ -250,7 +250,7 @@ public void TestDoubleSentence2Classes()
             var dataPrepTransformer = dataPrep.Fit(dataView);
             var preppedData = dataPrepTransformer.Transform(dataView);
 
-            var estimator = ML.MulticlassClassification.Trainers.TextClassification(outputColumnName: "outputColumn", sentence1ColumnName: "Sentence", sentence2ColumnName: "Sentence2", validationSet: preppedData)
+            var estimator = ML.MulticlassClassification.Trainers.TextClassification(2, outputColumnName: "outputColumn", sentence1ColumnName: "Sentence", sentence2ColumnName: "Sentence2", validationSet: preppedData)
```

Comment:
>2 [](http://example.com/codeflow?start=84&length=1)

will we add later a test for a different number of classes?

Reply:
I have a test for 3 classes already, but since 2 was the default I only had to add it in here as the test with 3 classes already had 3 specified.